libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -83,6 +83,14 @@
         EDITOR_TAPPED_UNLINK, // Visual editor only
         EDITOR_TAPPED_UNORDERED_LIST, // Visual editor only
         ME_ACCESSED,
+        ME_GRAVATAR_TAPPED,
+        ME_GRAVATAR_TAPPED_TOOLTIP,
+        ME_GRAVATAR_PERMISSIONS_DENIED,
+        ME_GRAVATAR_PERMISSIONS_ACCEPTED,
+        ME_GRAVATAR_SHOT_NEW,
```

Comment:
Since we're not really tracking anything related to it right now - no - but it seems okay. Calypso web just launches out to Gravatar.com so this is more functionality. 😄